src/platforms/common/configuration/filtering.mdx

```diff
@@ -4,6 +4,7 @@ sidebar_order: 60
 description: "Learn more about how to configure your SDK to filter events reported to Sentry."
 notSupported:
   - perl
+  - kotlin-multiplatform
```

Comment:
What if we short that to `kmp` everywhere?

Reply:
Not sure how much this would impact seo and general visiblity, I'd assume that the whole word in the link would be better in that regard

Reply:
Good point, lets let the docs team review this.

Reply:
Let's keep kotlin-multiplatform